accepted/future-releases/enhanced-enums/feature-specification.md

```diff
@@ -97,14 +101,14 @@ The semantics of such an enum declaration, *E*, is defined as introducing a (sem
 
 - **Declared members**: For each member declaration of the `enum` declaration *E*, the same member is added to the class *C*. This includes constructors (which must be `const` generative or non-`const` factory constructors.)
 
-- **Default constructor**: If no generative constructors were declared, and no unnamed factory constructor was added,
+- **Default constructor**: If no generative constructors were declared, and also no unnamed factory constructor was declared,
```

Comment:
The language specification considers the name of a constructor like `class C { C(); }` to be `C`, and the name of `class C { C.named(); }` to be `C.named`, for all kinds of constructors including factories. This means that we don't have to struggle with "unnamed" constructors. It doesn't hurt to introduce the name of the enum at this point, because it will be needed in line 108 anyway.

Reply:
I do think that it works better to say that the names of constructors are `C` and `C.named` rather than "the unnamed constructor declared by the class `C`" and "a constructor named `named` which is declared by the class `C`", even though I do recognize that "declared by the class `C`" can be obtained from the context in some cases.

However, let's get this landed with one or the other terminology, the first thing we need is to ensure consistency within any single document.

Reply:
👍